src/mudlet.cpp

```diff
@@ -687,244 +391,615 @@ mudlet::mudlet()
     auto consoleCloser = new TConsoleMonitor(mpDebugArea);
     mpDebugArea->installEventFilter(consoleCloser);
 
-    QSize generalRule(qApp->desktop()->size());
```

Comment:
I am not sure what is going here - the git diff code is getting confused and marking two (I think) identical sections as differences - I think it is missing that parts of a changed section which *does* have common lines is being seen as the bit that doesn't change... :man_shrugging: